browser/app/js/components/SideBar.js

```diff
@@ -14,86 +14,37 @@
  * limitations under the License.
  */
 
-import React from 'react'
-import classNames from 'classnames'
-import ClickOutHandler from 'react-onclickout'
-import Scrollbars from 'react-custom-scrollbars/lib/Scrollbars'
-import connect from 'react-redux/lib/components/connect'
+import React from "react"
+import classNames from "classnames"
+import ClickOutHandler from "react-onclickout"
+import { connect } from "react-redux"
 
-import logo from '../../img/logo.svg'
-import Dropdown from 'react-bootstrap/lib/Dropdown'
-
-let SideBar = ({visibleBuckets, loadBucket, currentBucket, selectBucket, searchBuckets, sidebarStatus, clickOutside, showPolicy, deleteBucket, toggleBucketDropdown, showBucketDropdown}) => {
-
-  const list = visibleBuckets.map((bucket, i) => {
-    return <li className={ classNames({
-                  'active': bucket === currentBucket
-                }) } key={ i } onClick={ (e) => selectBucket(e, bucket) }>
-             <a href="" className={ classNames({
-                                      'fesli-loading': bucket === loadBucket
-                                    }) }>
-               { bucket }
-             </a>
-             <Dropdown open={bucket === currentBucket && showBucketDropdown} onToggle={toggleBucketDropdown} className="bucket-dropdown" id="bucket-dropdown">
-               <Dropdown.Toggle noCaret>
-                 <i className="zmdi zmdi-more-vert" />
-               </Dropdown.Toggle>
-               <Dropdown.Menu className="dropdown-menu-right">
-                 <li>
-                   <a onClick={ showPolicy }>Edit policy</a>
-                 </li>
-                 <li>
-                   <a onClick={ (e) => deleteBucket(e, bucket) }>Delete</a>
-                 </li>
-               </Dropdown.Menu>
-             </Dropdown>
-           </li>
-  })
+import logo from "../../img/logo.svg"
+import Dropdown from "react-bootstrap/lib/Dropdown"
+import BucketSearch from "./BucketSearch"
+import BucketList from "./BucketList"
+import Host from "./Host"
 
+export const SideBar = () => {
   return (
-    <ClickOutHandler onClickOut={ clickOutside }>
-      <div className={ classNames({
-                         'fe-sidebar': true,
-                         'toggled': sidebarStatus
-                       }) }>
+    <ClickOutHandler>
```

Comment:
Do we need the onClickOut attribute here? Currently, when I click outside the sidebar, I get a message in the console saying `onClickOut is not defined`. We might have to implement this change later on as `onClickOut` handled hiding the sidebar when the user clicked outside the sidebar on mobile view (when the page width was less than 992px).

Reply:
yes, onClickOut is used to close the sidebar on mobile. I am working on the next PR which provides open/close sidebar and added the handler for onClickOut.